README.md

```
@@ -89,9 +89,9 @@ python main.py --port 9000 --datadir ./node1_data
 *Note: Keep this terminal open to interact with the node via the CLI.*
 
 ### 2. Connecting to an Existing Chain
-To connect a secondary node to the network, start a new instance on a different port and point it to the seed node using the `--connect` flag.
+To connect a secondary node to the network, start a new instance on a different port and point it to the seed node using the `--connect` flag with the full multiaddress the seed node printed on startup (it includes the seed's peer ID, not just its host and port).
 ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)